components/Downloads/ChangelogModal/index.module.css

```diff
@@ -0,0 +1,71 @@
+/* What color should the svg be? */
+svg {
+  @apply h-4
+    w-4;
+}
+
+a {
+  @apply w-fit
+    underline;
+}
+
+.dialogOverlay {
+  @apply fixed
+    inset-0
+    bg-white
+    bg-opacity-90
+    backdrop-blur-lg
+    dark:bg-neutral-900
+    dark:bg-opacity-80;
+}
+
+.dialogContent {
+  @apply fixed
+    left-[50%]
+    top-[40%]
+    inline-flex
+    max-h-[85vh]
+    w-3/4
+    translate-x-[-50%]
+    translate-y-[-50%]
+    flex-col
+    rounded
+    border
+    border-neutral-200
+    bg-white
+    p-12
+    focus:outline-none
+    dark:bg-neutral-900;
+}
+
+.dialogTitle {
+  @apply /*
+    Had trouble figuring out how other components were just inheriting fontfamily from
```

Comment:
Right, no need to define `font-open-sans` here, it's already being defined on Storybook (preview.tsx) https://github.com/nodejs/nodejs.org/blob/main/.storybook/preview.tsx#L37